workers/unity/Assets/Fps/Scripts/SetupLogic/ConnectionStateReporter.cs

```diff
@@ -0,0 +1,98 @@
+﻿using UnityEngine;
+
+namespace Fps
+{
+    public static class ConnectionStateReporter
+    {
+        public enum State
+        {
+            None,
+            GettingDeploymentList,
+            DeploymentListAvailable,
+            FailedToGetDeploymentList,
+            Connecting,
+            Connected,
+            ConnectionFailed,
+            Spawning,
+            Spawned,
+            SpawningFailed,
+            WorkerDisconnected
+        }
+
+        public static bool AreConnected { get; private set; }
```

Comment:
`IsConnected`? or do we actually mean multiple connections?

Reply:
"We" are connected, of course :P "we" being the computer and the user...